packages/admin/dashboard/src/i18n/translations/fr.json

```diff
@@ -0,0 +1,2748 @@
+				"customer": "Clients",
+				"customerGroup": "Groupes de clients",
+				"product": "Produits",
+				"productVariant": "Variantes de produits",
+				"inventory": "Inventaire",
+				"reservation": "Réservations",
+				"category": "Catégories",
+				"collection": "Collections",
+				"order": "Commandes",
+				"promotion": "Promotions",
+				"campaign": "Campagnes",
+				"priceList": "Listes de prix",
+				"user": "Utilisateurs",
+				"region": "Régions",
+				"taxRegion": "Régions fiscales",
+				"returnReason": "Motifs de retour",
+				"salesChannel": "Canaux de vente",
+				"productType": "Types de produits",
+				"productTag": "Tags de produits",
+				"location": "Emplacements",
+				"shippingProfile": "Profils d'expédition",
+				"publishableApiKey": "Clés API publiques",
+				"secretApiKey": "Clés API secrètes",
+				"command": "Commandes",
+				"navigation": "Navigation"
+			}
+		},
+		"keyboardShortcuts": {
+			"pageShortcut": "Sauter à",
+			"settingShortcut": "Paramètres",
+			"commandShortcut": "Commandes",
+			"then": "alors",
+			"navigation": {
+				"goToOrders": "Commandes",
+				"goToProducts": "Produits",
+				"goToCollections": "Collections",
+				"goToCategories": "Catégories",
+				"goToCustomers": "Clients",
+				"goToCustomerGroups": "Groupes de clients",
+				"goToInventory": "Inventaire",
+				"goToReservations": "Réservations",
+				"goToPriceLists": "Listes de prix",
+				"goToPromotions": "Promotions",
+				"goToCampaigns": "Campagnes"
+			},
+			"settings": {
+				"goToSettings": "Settings",
+				"goToStore": "Boutique",
+				"goToUsers": "Utilisateurs",
+				"goToRegions": "Régions",
+				"goToTaxRegions": "Régions fiscales",
+				"goToSalesChannels": "Canaux de vente",
+				"goToProductTypes": "Types de produits",
+				"goToLocations": "Emplacements",
+				"goToPublishableApiKeys": "Clés API publiques",
+				"goToSecretApiKeys": "Clés API secrètes",
+				"goToWorkflows": "Workflows",
+				"goToProfile": "Profil",
+				"goToReturnReasons": "Motifs de retour"
+			}
+		},
+		"menus": {
+			"user": {
+				"documentation": "Documentation",
+				"changelog": "Changelog",
+				"shortcuts": "Raccourcis",
+				"profileSettings": "Paramètres du profil",
+				"theme": {
+					"label": "Thème",
+					"dark": "Dark",
+					"light": "Light",
+					"system": "Système"
+				}
+			},
+			"store": {
+				"label": "Boutique",
+				"storeSettings": "Paramètres de la boutique"
+			},
+			"actions": {
+				"logout": "Déconnexion"
+			}
+		},
+		"nav": {
+			"accessibility": {
+				"title": "Navigation",
+				"description": "Menu de navigation pour le tableau de bord."
+			},
+			"common": {
+				"extensions": "Extensions"
+			},
+			"main": {
+				"store": "Boutique",
+				"storeSettings": "Paramètres de la boutique"
+			},
+			"settings": {
+				"header": "Settings",
+				"general": "General",
+				"developer": "Développeur",
+				"myAccount": "Mon compte"
+			}
+		}
+	},
+	"dataGrid": {
+		"columns": {
+			"view": "View",
```

Comment:
```suggestion
			"view": "Vue",
```

wdyt?

Reply:
Depending on the context maybe "Affichage" is more suitable

Reply:
Oh yeah, that seems better indeed 💪